Modules/socketmodule.c

```diff
@@ -7883,5 +7946,10 @@ PyInit__socket(void)
 #if defined(USE_GETHOSTBYNAME_LOCK) || defined(USE_GETADDRINFO_LOCK)
     netdb_lock = PyThread_allocate_lock();
 #endif
+
+#ifdef MS_WINDOWS
+    return remove_unusable_flags(m);
```

Comment:
Make this `remove_unusable_flags(m)` and then leave the `return` outside the `if` block (it's easier to reason about an optional action than a potential transformation, and the same object is always returned)